tests/strategies/test_strategy_base.py

```diff
@@ -449,19 +449,19 @@ def test_post_offer_with_replace_existing(market_class):
 
     # Post a first offer on the market
     offer_1_args = {
-        "price": 1, "energy": 1, "seller": "seller-name", "seller_origin": "seller-origin-name"}
+        "price": 1, "energy": 1, "seller": "FakeOwner", "seller_origin": "seller-origin-name"}
```

Comment:
Sure. The seller/buyer name should match the owner name [here](https://github.com/gridsingularity/gsy-e/blob/9fc1f45a9491cb56b99072ee79e6f5c53461b85b/src/gsy_e/models/strategy/__init__.py#L509) and [here](https://github.com/gridsingularity/gsy-e/blob/9fc1f45a9491cb56b99072ee79e6f5c53461b85b/src/gsy_e/models/strategy/__init__.py#L752), otherwise we end up having failed assertions.

Reply:
Ah, I see now. Thanks! Thanks for adding the assertions"